src/examples/SingleSignOn/SingleSignOn.js

```diff
@@ -0,0 +1,132 @@
+import * as FormioLoader from "../../matrixHelpers/FormioLoader";
+
+export const createSSoForm = () => {
+  const div = document.createElement("div");
+  div.innerHTML = `
+  <div id="oidc_form"></div>
+  <div id="formio"></div>
+  `;
+
+  const oidcform = div.querySelector("#oidc_form");
+  const formioDiv = div.querySelector("#formio");
+
+  function appendSpinner(parentElement) {
+    parentElement.innerHTML = `<img src="https://www.qld.gov.au/__data/assets/image/0019/126703/Spinner-1s-200px.png"/>`;
+  }
+
+  setTimeout(() => {
+    const namespace = "formio_ncwawujlwylhrfy";
```

Comment:
Perhaps define the project ID first and use it to assemble this value? Otherwise it looks random at first.